### DISCUSSION TITLE
script ar

### DISCUSSION THREAD

**Discussion by zjafarpishe** [google/model-viewer#4374] [Q&A]
Augmented reality from the <script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.1.1/model-viewer.min.js"></script>  link is different from the site's   augmented reality https://modelviewer.dev/examples/augmentedreality/index.html  I also need the site itself to show the material in its entirety

**Comment by elalish** [comment]
That example shows a bunch of HTML and CSS that customizes the WebXR AR experience, so you would need to add something similar to your page to see something similar. 

What material is missing?
